src/main/java/seedu/address/logic/parser/AddContactToTeamCommandParser.java

```diff
@@ -0,0 +1,70 @@
+//package seedu.address.logic.parser;
+//
+//import seedu.address.logic.commands.AddContactToTeamCommand;
+//import seedu.address.logic.parser.exceptions.ParseException;
+//
+//import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+//
+///**
+// * Parses input arguments and creates a new AddContactToTeamCommand object.
+// */
+//public class AddContactToTeamCommandParser implements Parser<AddContactToTeamCommand> {
+//
+//    @Override
+//    public AddContactToTeamCommand parse(String args) throws ParseException {
+//        String[] indices = args.trim().split("\\s+");
+//        if (indices.length != 4 || !indices[0].equals("team") || !indices[2].equals("add")) {
+//            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddContactToTeamCommand.MESSAGE_USAGE));
+//        }
+//
+//        int teamIndex;
+//        int contactIndex;
+//        try {
+//            teamIndex = Integer.parseInt(indices[1]);
+//            contactIndex = Integer.parseInt(indices[3]);
+//        } catch (NumberFormatException e) {
+//            throw new ParseException("Invalid index format");
+//        }
+//
+//        return new AddContactToTeamCommand(teamIndex, contactIndex);
+//    }
+//
+//}
+
+package seedu.address.logic.parser;
+
+import seedu.address.logic.commands.AddContactToTeamCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+/**
+ * Parses input arguments and creates a new AddContactToTeamCommand object.
+ */
+public class AddContactToTeamCommandParser implements Parser<AddContactToTeamCommand> {
+
+    private static final String COMMAND_WORD = "add-to-team";
+
+    @Override
+    public AddContactToTeamCommand parse(String args) throws ParseException {
```

Comment:
This is called from `TeamCommandsParser`, which parses the team index and contact index (or other args) as seperate arguments, so you should have two arguments for this method.
```suggestion
    public AddContactToTeamCommand parse(String teamIndex, String contactIndex) throws ParseException {
```